scripts/qa_crowbarsetup.sh

```diff
@@ -2609,6 +2609,13 @@ function custom_configuration
             if [ $networkingplugin = openvswitch ] ; then
                 if [[ $networkingmode = vxlan ]] || iscloudver 6plus; then
                     proposal_set_value neutron default "['attributes']['neutron']['ml2_type_drivers']" "['vxlan','vlan', 'gre']"
+                    if [[ $want_l3_ha = 1 ]]; then
```

Comment:
I think this needs to be outside the networkingplugin = openvswitch block because it is unrelated to the L2 driver implmeentation (its an L3 feature which can be combined with different L2 technologies, at least in principle)

Reply:
urgs. sorry. missed that. fixed